Mathlib/Data/Prod/PProd.lean

```diff
@@ -0,0 +1,43 @@
+/-
+Copyright (c) 2020 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.Logic.Basic
+
+/-!
+# Extra facts about `pprod`
+-/
+
+
+open Function
+
+variable {α β γ δ : Sort _}
+
+namespace PProd
+
+@[simp]
+theorem mk.eta {p : PProd α β} : PProd.mk p.1 p.2 = p :=
+  PProd.casesOn p fun _ _ => rfl
+
+@[simp]
+theorem «forall» {p : PProd α β → Prop} : (∀ x, p x) ↔ ∀ a b, p ⟨a, b⟩ :=
+  ⟨fun h a b => h ⟨a, b⟩, fun h ⟨a, b⟩ => h a b⟩
+
+@[simp]
+theorem «exists» {p : PProd α β → Prop} : (∃ x, p x) ↔ ∃ a b, p ⟨a, b⟩ :=
+  ⟨fun ⟨⟨a, b⟩, h⟩ => ⟨a, b, h⟩, fun ⟨a, b, h⟩ => ⟨⟨a, b⟩, h⟩⟩
+
+theorem forall' {p : α → β → Prop} : (∀ x : PProd α β, p x.1 x.2) ↔ ∀ a b, p a b :=
+  PProd.forall
+
+theorem exists' {p : α → β → Prop} : (∃ x : PProd α β, p x.1 x.2) ↔ ∃ a b, p a b :=
+  PProd.exists
+
+end PProd
+
+theorem Function.injective.pprod_map {f : α → β} {g : γ → δ} (hf : injective f) (hg : injective g) :
```

Comment:
```suggestion
theorem Function.Injective.pprod_map {f : α → β} {g : γ → δ} (hf : injective f) (hg : injective g) :
```

Reply:
Sorry, I don't understand -- I thought we're keeping it as `def injective` lowercase.

Reply:
I think we will need to change the casing on the `hf` and `hg` also.